tests/infra_test.go

```diff
@@ -665,117 +669,116 @@ var _ = Describe("Infrastructure", func() {
 				Transport: tr,
 			}
 
-			errors := make(chan error)
-			for ix := 0; ix < concurrency; ix++ {
-				go func(ix int) {
-					req, _ := http.NewRequest("GET", metricsURL, nil)
-					resp, err := client.Do(req)
+			for _, url := range metricsURLs {
+				if urlIsIPv6(url) {
+					By("Skip Scraping the Prometheus endpoint (IPv6) until issue https://github.com/kubevirt/kubevirt/issues/4145 is fixed")
+					continue
+				}
+
+				By(fmt.Sprintf("Scraping the Prometheus endpoint %s", url))
+				errors := make(chan error)
+				for ix := 0; ix < concurrency; ix++ {
+					go func(ix int) {
+						req, _ := http.NewRequest("GET", url, nil)
+						resp, err := client.Do(req)
+						if err != nil {
+							fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
+						} else {
+							resp.Body.Close()
+						}
+						errors <- err
+					}(ix)
+				}
+
+				errorCount := 0
+				for ix := 0; ix < concurrency; ix++ {
+					err := <-errors
 					if err != nil {
-						fmt.Fprintf(GinkgoWriter, "client: request: %v #%d: %v\n", req, ix, err) // troubleshooting helper
-					} else {
-						resp.Body.Close()
+						if urlErr, ok := err.(*netURL.Error); ok {
```

Comment:
What is this type `netURL.Error`?

Reply:
this is what 
https://github.com/kubevirt/kubevirt/pull/4140/files/e1d791ac31e99073b616f69ba29540a4f1f794eb#diff-29539fc2ce2beadde4ecbf90b4a00816R683
returns (netUrl is actually "net/url", just the word url was busy)

and in the struct of it, it has a string of the error, that can shows us if its the bad or good errors

same cast happen on other files of kubevirt and its vendored folders